### ISSUE TITLE
Archiving repository

### ISSUE THREAD

**Issue by aalvarado** [aalvarado/JiraDarkTheme#30]
I started trying out DarkReader and seems to do a better job rotating colors on all pages, not just the ones specified in this style. 

This is in contrast of how I usually work with stylus. The lack of a clear semantic html structure in modern web apps prevents people from targeting certain elements and restyling them. Since most of the React apps use generated clases, this becomes impossible to maintain.

In short, this style is incomplete in different places, Dark Reader does a better job which I've switched to and been using for several days now

**Comment by aalvarado** [comment]
I want to have stats on this repo but archiving it removes that. I've disabled this and seems like there are still people starring and forking.
